plugins/gin/intercepter.go

```diff
@@ -70,3 +77,26 @@ func isFirstHandle(c interface{}) bool {
 	}
 	return true
 }
+
+func collectHttpHeaders(span tracing.Span, requestHeaders http.Header) {
+	if len(runtimeConfig.CollectHttpHeaders) == 0 {
```

Comment:
If len==0 then split? Do you mis-write this?

Reply:
You should update the gin plugin test to verify this feature.

Reply:
> If len==0 then split? Do you mis-write this?

They are different config. I use runtimeConfig to store header list, cause the config can't accept array type config param.

Reply:
> You should update the gin plugin test to verify this feature.

right